airflow/kubernetes/worker_configuration.py

```diff
@@ -28,7 +28,11 @@
 
 
 class WorkerConfiguration(LoggingMixin):
-    """Contains Kubernetes Airflow Worker configuration logic"""
+    """
+    Contains Kubernetes Airflow Worker configuration logic
+    :param kube_config: the kubernetes configuration from airflow.cfg
```

Comment:
```suggestion

        :param kube_config: the kubernetes configuration from airflow.cfg
```
Without this enter the documentation will not build properly.